examples/opensora_hpcai/scripts/train.py

```diff
@@ -389,10 +474,30 @@ def main(args):
             start_epoch=start_epoch,
             model_name="STDiT",
             record_lr=False,
+            integrated_save=False,
```

Comment:
Is the ckpt saving working properly under optimizer parallel mode? when `parallel_mode="semi"` and `use_sequence_parallelism=False`.

Reply:
yes. They are actually under the same parallel mechanism

Reply:
moved optimizer parallel algo to the sequence_parallel callback branch.